collectors/ebpf.plugin/ebpf_cachestat.c

```diff
@@ -1002,8 +1296,9 @@ void *ebpf_cachestat_thread(void *ptr)
                        cachestat_counter_dimension_name, cachestat_counter_dimension_name,
                        algorithms, NETDATA_CACHESTAT_END);
 
+    pthread_mutex_lock(&lock);
```

Comment:
BTW, do you plan to use libuv threading and synchronization primitives in the eBPF plugin?

Reply:
This is a change that we cannot discard, mainly considering that [Microsoft](https://github.com/microsoft/ebpf-for-windows) has working to improve eBPF support. Considering that we are right now working to finish the CO-RE integration, this could be the next work with plugin (ping @cpipilas ).